eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/util/TraceUtils.java

```diff
@@ -106,18 +100,21 @@ public static void finishSpan(ChannelHandlerContext ctx, CloudEvent event) {
     public static void finishSpanWithException(ChannelHandlerContext ctx, CloudEvent event,
         String errMsg, Throwable e) {
         try {
-            log.debug("finishSpanWithException with event:{}", event);
+            if (log.isWarnEnabled()) {
+                log.warn("finishSpanWithException with event:{}", event);
+            }
             EventMeshServer.getTrace().addTraceInfoToSpan(ctx, event);
             EventMeshServer.getTrace().finishSpan(ctx, StatusCode.ERROR, errMsg, e);
         } catch (Throwable ex) {
             log.error("upload trace fail when finishSpanWithException", ex);
         }
     }
 
-    public static void finishSpanWithException(Span span, Map<String, Object> map, String errMsg,
-        Throwable e) {
+    public static void finishSpanWithException(Span span, Map<String, Object> map, String errMsg, Throwable e) {
         try {
-            log.debug("finishSpanWithException with map:{}", map);
+            if (log.isWarnEnabled()) {
+                log.warn("FinishSpanWithException with map:{}", map);
+            }
```

Comment:
May I ask that why logging level should be upgraded to `warn` here?

By the way, `log.isWarnEnabled()` is redundant~